src/DacpacTool/PackageBuilder.cs

```diff
@@ -112,19 +112,16 @@ public bool ValidateModel()
                 if (modelError.Severity == ModelErrorSeverity.Error)
                 {
                     validationErrors++;
+                    Console.WriteLine(modelError.GetOutputMessage(modelError.Severity));
```

Comment:
Why is this part of this PR?

Reply:
Because of I and my team actively use this great project in our development. Using NuGet packages is what really amazing feature. But we need some features witch presence in SSDT projects and are absent in SqlProj. So I decide to add those features to SqlProj. But approving of them is too slow. All of my additions was developed in two days. As @jmezach ask to split them to partials PR. I do It. but every approve take more than 2 weeks to proceed..... So I think, that It will be more productive to us to maintain our fork separately from the main project. Sorry, I may be newbie for git and GitHub, I've using tfvc for many years, and vss before it. And I began to learn git about some months ago. So switching between branches, rebasing, pull requesting and so, and so, is too hard for me (for now).